lib/protos/jellyfish/server_notifications.pb.ex

```diff
@@ -121,6 +121,22 @@ defmodule Jellyfish.ServerMessage.HlsPlayable do
   field :component_id, 2, type: :string, json_name: "componentId"
 end
 
+defmodule Jellyfish.ServerMessage.HlsUploaded do
```

Comment:
Imo `HlsUploaded` may not be so descriptive. Perhaps we should name this `HlsUploadSuccess` to indicate directly that the upload was a success

Reply:
I think that `HlsUploaded` is a descriptive EVENT as `RoomCreated`, adding `Success` is redundant and incompatible with the rest of messages.

What do you think @Rados13

Reply:
IMO, it is descriptive enough.